recipe/meta.yaml

```diff
@@ -8,14 +9,21 @@ package:
 
 source:
   fn: SuiteSparse-{{ version }}.tar.gz
-  url: http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-{{ version }}.tar.gz
-  sha256: 698b5c455645bb1ad29a185f0d52025f3bd7cb7261e182c8878b0eb60567a714
+  url: http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-{{ version }}.tar.gz  # [not win]
+  sha256: 698b5c455645bb1ad29a185f0d52025f3bd7cb7261e182c8878b0eb60567a714  # [not win]
+  url: https://github.com/jlblancoc/suitesparse-metis-for-windows/archive/{{ version_win }}.tar.gz  # [win]
+  sha256: 719065607201f445d490b6bb7bb43df2e8f1affb187551c841d5d2a3dd72f2e7  # [win]
+  patches:
+    - gh_53.patch  # [win]
+    - gh_54.patch  # [win]
 
 build:
-  skip: true  # [win]
+  skip: true  # [win32]
```

Comment:
Just out of curiosity, what are the challenges with Windows 32-bit?

Reply:
There is no 64-bit OpenBLAS available on any platform.

Reply:
How does this build on Linux and OSX, since this package requires OpenBLAS.

Reply:
What do you mean? The last version of macOS to support 32-bit was 10.6, which has never been supported in conda-forge and was only long ago supported by Anaconda. Similarly we don't even have images for building 32-bit Linux packages. Windows is the only platform conda-forge supports 32-bit on. Hence the question.

If you are saying there is no 32-bit build of `openblas` for Windows, that's all I was looking for.

Reply:
That's correct. There's no 32 bit OpenBLAS for windows because there's no 32 bit Fortran compiler for Windows. I could make a 32 bit Fortran compiler for windows, but would the benefits really be worth it? I thought about this, and I'm not really sure this would provide a significant benefit to anyone.

Reply:
I think you're right. It's probably not worth it.